Puppetfile

```diff
@@ -9,6 +9,7 @@ mod 'theforeman/dhcp',          :git => 'https://github.com/theforeman/puppet-dh
 mod 'theforeman/dns',           :git => 'https://github.com/theforeman/puppet-dns'
 mod 'theforeman/git',           :git => 'https://github.com/theforeman/puppet-git'
 mod 'theforeman/tftp',          :git => 'https://github.com/theforeman/puppet-tftp'
+mod 'puppetlabs/apache',        :git => 'https://github.com/puppetlabs/puppetlabs-apache.git'
```

Comment:
Could you add a note that we're pulling this in because of a certain commit and/or refer to the Puppetlabs JIRA issue? Makes it easier to check in the future if it can be removed.